### PR TITLE
Add icons for Avro and Parquet Plugins

### PR DESCRIPTION
**Pull request by ecfm**
ORC plugin is temporarily removed in this release until its HIVE explore issue is fixed

### CODE REVIEW

**Comment by bdmogal** [comment]
Instead of this, create an icons directory at the top level, and add the png files there. Then `cdap-maven-plugin` will generate this widget json

File: widgets/AvroDynamicPartitionedDataset-batchsink.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "metadata": {
     "spec-version": "1.3"
   },
+  "display-name": "Avro Dynamic Partitioned Dataset",
```
Comment (bdmogal) [inline widgets/AvroDynamicPartitionedDataset-batchsink.json:5]: If this is dataset, the other one should also be dataset?

Comment (ecfm) [reply widgets/AvroDynamicPartitionedDataset-batchsink.json:5]: Not sure which is a better name. Their parent class is `PartitionedFileSetSink` but both of their names end with Dataset

Comment (bdmogal) [reply widgets/AvroDynamicPartitionedDataset-batchsink.json:5]: I think Dataset is better, its more well known

**Review by bdmogal — APPROVED** [review]
squash and merge.

**Pull request merged**